Essentials/src/messages.properties

```diff
@@ -508,8 +489,8 @@ sudoRun=\u00a76Forcing\u00a7c {0} \u00a76to run\:\u00a7r /{1}
 suicideMessage=\u00a76Goodbye cruel world...
 suicideSuccess=\u00a76Player \u00a7c{0} \u00a76took their own life.
 survival=survival
-takenFromAccount=\u00a7a{0} has been taken from your account.
-takenFromOthersAccount=\u00a7a{0} taken from {1}\u00a7a account. New balance\: {2}.
+takenFromAccount=\u00a7e{0}\u00a7a has been taken from your account.
+takenFromOthersAccount=\u00a7e{0}\u00a7a taken from\u00a7e {1}\u00a7a account. New balance\:\u00a7e {2}
```

Comment:
I still think the period meets with the rest of the message styling

Reply:
Yes, but don't have the same color.

Reply:
"New balance: $50" isn't a sentence, so I'd say it doesn't matter whether the period is there or not.